src/main/markdown/eco/com-and-more.md

```diff
@@ -0,0 +1,57 @@
+# Communication, Validation & more ... 
+
+GWT comes with a variate of frameworks to communicate with the server.
+Take a look at [Ajax Communication](/doc/latest/tutorial/clientserver.html) for more information about modules GWT offers.
+Besides that, there are many useful libraries.
+
+Check these:
+
+## Client-Server-Communication
+
+Besides the [communication to the server provided by GWT](/doc/latest/tutorial/clientserver.html) 
+there are alternative implementations, too.
+
+### domino-rest<a id="domino-rest"></a>
+
+Domino-rest is a lib for generating rest clients from JaxRs compatible interfaces, and the generated clients can be used 
+from both client side -browser- using GWT and on the server. The lib works with GWT2 and also with GWT3/J2CL. The 
+serialization and deserialization is based on domino-jackson and service definition is based on JaxRs annotations, while 
```

Comment:
a link to domino-jackson documentation website

https://dominokit.com/solutions/domino-jackson/v1